pages/index.tsx

```diff
@@ -86,7 +87,16 @@ export default function Home({
 
 export const getServerSideProps: GetServerSideProps = async ({ req, locale }) => {
   const isProd = process.env.NODE_ENV === 'production'
```

Comment:
what do you think about replacing `isProd` with something that isn't ambiguous? since test, staging, etc Azure envs are all "isProd". something like `isAzureEnv` or `isNotLocalEnv`